code/datums/ai/dog/dog_controller.dm

```diff
@@ -75,6 +75,9 @@
 	else if(DT_PROB(10, delta_time))
 		living_pawn.manual_emote(pick("dances around.", "chases [living_pawn.p_their()] tail!</span>"))
 		living_pawn.AddComponent(/datum/component/spinny)
+		for (var/mob/i in oviewers(living_pawn))
+			if (ishuman(i) && i.mind) //if viewer is human and a player. do non-human players have moods?
```

Comment:
```suggestion
		for(var/mob/living/carbon/human/H in oviewers(living_pawn))
			if(H.mind)
```
All mob/living can have mind, what you're doing here is giving the moodlet to only humans
Oh and that for look has istype built into it